Documentation/Signoffs/Line-Sensing-Signoff.md

```diff
@@ -0,0 +1,118 @@
+* C10: The robot shall turn 360 degrees left and right and move forwards and backwards based on sensor inputs.
+* C11: The robot shall have a navigation system that controls movement.
+	* The line sensing subsystem will transmit data to the master control telling it where the robot is with respect to the sensed line.
+		
+- C12: The robot shall know its location within a two-inch tolerance.
+	- This constraint holds the line sensing subsystem to a maximum of 2-inch tolerance. If the team is using the line sensing option for traversing the track, the two-inch tolerance will be calculated from the center of the desired line.
+
+### Derived from Broader Implications
+	
+- C14: The robot shall have a user manual that explains all functions and capabilities of the robot. 
+	- This constraint will require all detailed design decisions to be documented and explained. Not only will the design have to be explained, but the motives behind the design and the other options explored so that future SECON teams have an understanding of why the design was created in a certain manner.
+		
+	
+	
+## Schematics
+
+### System Schematics
+
+![Alt text](https://github.com/lchapman42/Control-Sensing-Wireless-Charging-Robot/blob/main/Documentation/Images/Line-Following/Line_Sensor_Connections.png)
+The schematic of the system is shown above. There will be two reflectance sensor arrays used. 
+    The Pololu QTR-8RC module has a perforation in the board that allows for two of the eight sensors to be broken
+	off. This leaves one reflectance sensor array with two LED emitters and receivers and one reflectance sensor array with six LED emitters
+	receivers. 
+	The array with six LED emitter/receiver pairs will be used at the head of the robot for 
+	forward movement about the line.
+	The array with two emitters and receivers will be used at the back of the robot for reversing. Because reversing is most likely done
+	in small intervals, the smaller module will suffice. Reversing will never be the first robot movement, so the front sensor set will be active
+	before the smaller module. This means that the robot will be centered in reference to the line before moving backwards. As a result, the smaller module
+	will be inately centered, allowing for less precision.
+
+The sensors will be powered by a 5 V regulated source coming from the power distribution subsystem. The sensor
+	outputs will be read by an Arduino Mega. The digital pins 22-28, 30 will be used to conserve analog and PWM pins for future sensor implementation.
+	
+<p align = "center">
+<img src = "https://github.com/lchapman42/Control-Sensing-Wireless-Charging-Robot/blob/main/Documentation/Images/Line-Following/2021_SECON_Board.png?raw=true"alt="Secon Board"/>
+</p>
+
+The last IEEE SECON Hardware competition arena with a line was in 2021. A small white line of 1" was used.
+	The Pololu QTR-8RC module spans 2.95" without the separation of the 6 RC and 2 RC modules. Because the width of the module is larger
```

Comment:
I am a little confused, 6RC and 2RC are responsible for forward and backward sensing, why the module width is larger than the line matter? I feel like if they are responsible for left and right movement, it would make more sense

Reply:
The 2 modules are responsible for detecting the right and left movement. The forward and reverse movements I was referring to was to differentiate the larger and smaller modules. The larger module will be mounted on the front of the robot to detect left and right movement while going forward, the smaller module will be mounted on the back of the robot to detect left and right movement while in reverse.

I will clarify further in the document. 
Thank you!

Reply:
Okay. That makes sense now. I can approve the signoff now.